web/src/routes/(user)/people/[personId]/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -317,8 +317,9 @@
             !person.isHidden,
         )
         .slice(0, 3);
-      await handleMergeSuggestion();
-      return;
+      if ((await handleMergeSuggestion()) !== false) {
```

Comment:
The semantics around this method and its return are a bit hard to follow. What do think about changing the return type from `Promise<false | undefined>` to `Promise<{ merged: boolean }>` and then the code looks like this:
```ts
const { merged } = await handleMergeSuggestion();
if(merged) {
  return;
}

await changeName();
```

Just replace all the `return;`s with `return { merged: false }`;